src/app/api/webhooks/clerk/route.ts

```
@@ -0,0 +1,134 @@
+      case "user.deleted":
+        if (eventData.id) {
+          try {
+            // Attempt to delete the user.
+            // This will fail if there are related records that prevent deletion due to foreign key constraints,
+            // unless cascading deletes are set up appropriately in the Prisma schema.
+            // It will also fail if the user does not exist (e.g., already deleted or never synced).
+            await db.user.delete({
+              where: { id: eventData.id },
+            });
+            console.log(`User ${eventData.id} deleted from local DB via webhook.`);
+          } catch (error: any) {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)